tools/cli/admin_dlq_commands.go

```diff
@@ -218,22 +223,27 @@ func AdminGetDLQMessages(c *cli.Context) {
 			}
 			pageToken = resp.NextPageToken
 		}
-		return rows
+		return rows, nil
 	}
 
 	table := []DLQRow{}
 	for shardID := range getShards(c) {
 		if remainingMessageCount <= 0 {
 			break
 		}
-		table = append(table, readShard(shardID)...)
+		tablesInShard, err := readShard(shardID)
+		if err != nil {
+			return err
```

Comment:
probably worth wrapping this error with some more details, it'll be hard to tell which one failed otherwise

Reply:
fixed